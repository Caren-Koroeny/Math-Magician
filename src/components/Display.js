import PropTypes from 'prop-types';

const Display = ({ total, operation, next }) => (
  <div className="displayscreen-count-section">
    <span className="input-screen">
      {total}
      {operation}
      {next}
    </span>

  </div>
);

Display.propTypes = {
  total: PropTypes.number,
  operation: PropTypes.string,
  next: PropTypes.string,
};
Display.defaultProps = {
  total: 0,
  operation: null,
  next: null,
};

export default Display;
