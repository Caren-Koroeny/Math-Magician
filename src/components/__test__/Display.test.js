import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../Display';

it('renders DisplayScreen correctly', () => {
  const tree = renderer.create(<Display />);
  expect(tree).toMatchSnapshot();
});
