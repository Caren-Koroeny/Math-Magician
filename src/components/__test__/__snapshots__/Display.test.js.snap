// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DisplayScreen correctly 1`] = `
<div
  className="displayscreen-count-section"
>
  <span
    className="input-screen"
  >
    0
  </span>
</div>
`;
